confd/templates/production.tmpl

```diff
@@ -295,88 +295,125 @@
+				  ]
 		  }
 	  },
       "PLANET": {
         "inActive": {{ getv "/ansyn/overlay/planet/inactive" "false" }},
         "treeViewItem": {
-        "text": "Planet",
-        "value": 1,
-            "collapsed": true,
-            "children": [
-              {
-                "text": "REScene",
-                "value": {
-                  "sensorType": "REScene"
-                }
-              },
-              {
-                "text": "PSScene3Band",
-                "value": {
-                  "sensorType": "PSScene3Band"
-                }
-              },
-              {
-                "text": "PSOrthoTile",
-                "value": {
-                  "sensorType": "PSOrthoTile"
-                }
-              },
-              {
-                "text": "REOrthoTile",
-                "value": {
-                  "sensorType": "REOrthoTile"
-                }
-              },
-              {
-                "text": "SkySatScene",
-                "value": {
-                  "sensorType": "SkySatScene"
-                }
-              },
-              {
-                "text": "Landsat8L1G",
-                "value": {
-                  "sensorType": "Landsat8L1G"
-                }
-              },
-              {
-                "text": "Sentinel2L1C",
-                "value": {
-                  "sensorType": "Sentinel2L1C"
-                }
-              },
-			  {
-				  "text": "Others",
-				  "value": {
-					"sensorType": "others"
-				  }
-				}
-            ]
+			"text": "Planet",
+			"value": 1,
+				"collapsed": true,
+				"children": [
+				  {
+					"text": "REScene",
+					"value": {
+					  "sensorType": "REScene"
+					}
+				  },
+				  {
+					"text": "PSScene3Band",
+					"value": {
+					  "sensorType": "PSScene3Band"
+					}
+				  },
+				  {
+					"text": "PSOrthoTile",
+					"value": {
+					  "sensorType": "PSOrthoTile"
+					}
+				  },
+				  {
+					"text": "REOrthoTile",
+					"value": {
+					  "sensorType": "REOrthoTile"
+					}
+				  },
+				  {
+					"text": "SkySatScene",
+					"value": {
+					  "sensorType": "SkySatScene"
+					}
+				  },
+				  {
+					"text": "Landsat8L1G",
+					"value": {
+					  "sensorType": "Landsat8L1G"
+					}
+				  },
+				  {
+					"text": "Sentinel2L1C",
+					"value": {
+					  "sensorType": "Sentinel2L1C"
+					}
+				  },
+				  {
+					  "text": "Others",
+					  "value": {
+						"sensorType": "others"
+					  }
+					}
+				]
           }
         },
         "OPEN_AERIAL": {
           "inActive": {{ getv "/ansyn/overlay/open/aerial/inactive" "false" }},
           "treeViewItem": {
             "text": "OpenAerial",
             "value": 1,
-            "indeterminate": true,
-            "collapsed": true,
-            "children": [
-              {
-                "text": "UAV",
-                "value": {
-                  "sensorType": "UAV"
-                }
-              }
-            ]
+				"indeterminate": true,
```

Comment:
Cancel all the changes to OPEN_AERIAL

Reply:
I did the changes because that the object's nesting was wrong